cuda/private/templates/BUILD.cupti

```diff
@@ -11,15 +11,6 @@ cc_import(
     target_compatible_with = ["@platforms//os:windows"],
 )
 
-cc_library(
-    name = "cupti_headers",
```

Comment:
Please just rename it to `cupti_headers`

Reply:
And the `if_windows([` part in `cupti` target

Reply:
thanks, done